controller_manager/test/test_controller_manager.cpp

```diff
@@ -423,23 +423,28 @@ TEST_P(TestControllerUpdateRates, check_the_controller_update_rate)
   EXPECT_EQ(test_controller->get_update_rate(), ctrl_update_rate);
   const auto cm_update_rate = cm_->get_update_rate();
   const auto controller_update_rate = test_controller->get_update_rate();
-  const auto controller_factor = (cm_update_rate / controller_update_rate);
-  const auto expected_controller_update_rate =
-    static_cast<unsigned int>(std::round(cm_update_rate / static_cast<double>(controller_factor)));
 
   const auto initial_counter = test_controller->internal_counter;
-  for (size_t update_counter = 1; update_counter <= 10 * cm_update_rate; ++update_counter)
+  rclcpp::Time time;
```

Comment:
```suggestion
  rclcpp::Time time=rclcpp::Time(0);
```
sorry for this c++ noob question, is this ensured to be initialized otherwise?

Reply:
Hey @christophfroehlich ,

I believe it is initialized yes, but yes it can be explicit. This way it is better. Can you please add spaces to your suggestion?, so the formatting wouldn't complain.

Reply:
I've added this along with the linter issue

Reply:
great